ReText/table.py

```diff
@@ -0,0 +1,91 @@
+from ReText import globalSettings
+from PyQt5.QtCore import pyqtSignal, Qt
+from PyQt5.QtWidgets import QDialog, QVBoxLayout, QTabWidget, \
+	QDialogButtonBox, QWidget, QGridLayout, QLabel, QSpinBox
+
+
+class ClickableLabel(QLabel):
+	clicked = pyqtSignal()
```

Comment:
This signal is not actually used, so you can just use QLabel here.

Reply:
OK, I remove the code for the line 8.